app/Services/AdminService.php

```diff
@@ -13,4 +14,14 @@ public static function sendInvite($studentEmail)
 
         return "email has been sent";
     }
+
+    public function storeAdmin($data)
```

Comment:
I would revisit the naming of this method to something like: "createAdminUserInfo" or something descriptive like that.